src/components/collections-grid/shop-by-room.js

```diff
@@ -19,18 +18,26 @@ export const ShopByRoom = ({
   return (
     <div className={sty.ShopByRoom}>
       <div className={sty.headerText}>
-        <PrismicRichText field={subheader?.richText} />
+        <PrismicRichText
+          field={subheader?.richText}
+          components={{
+            paragraph: ({ children }) => (
+              <h2 className={sty.headerText}>{children}</h2>
+            ),
+          }}
+        />
       </div>
```

Comment:
If you're pulling in rich text here and putting it into an h2 is that going to cause problems? Is this so you can have italicized text?